src/cms/views/chat/chat_actions.py

```diff
@@ -20,6 +20,9 @@ def send_chat_message(request):
     """
     Send chat message
 
+    :param request: The current request
+    :type request: ~django.http.HttpResponse
```

Comment:
```suggestion
    :type request: ~django.http.HttpRequest
```
request and response are two different things :wink: